vendor/runner/runner.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const glob = require('glob');
+const uniq = require('lodash.uniq');
+
+/**
+ * Load a collection of ServiceTester objects and register them with Mocha.
+ */
+class Runner {
+  /**
+   * Function to invoke before each test. This is a stub which can be
+   * overridden on instances.
+   */
+  beforeEach () {}
+
+  /**
+   * Prepare the runner by loading up all the ServiceTester objects.
+   */
+  prepare () {
+    this.testers = glob.sync(`${__dirname}/../*.js`).map(require);
```

Comment:
I usually like explicitly using an arrow function for `map` to ensure it just passes the one argument I care about: `.map(name => require(name))`